buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -161,6 +161,27 @@ echo "ghprbTargetBranch:'${ghprbTargetBranch}'"
 ghprbActualCommit = (params.ghprbActualCommit) ? params.ghprbActualCommit : ""
 echo "ghprbActualCommit:'${ghprbActualCommit}'"
 
+// If custom repo/branch/refspec is passed, use it, elif build is OpenJ9 PR, use pr merge-ref/refspec, else master/blank for default respectively
+SCM_REPO = 'https://github.com/eclipse/openj9.git'
+if (params.SCM_REPO) {
+    SCM_REPO =params.SCM_REPO
+}
+echo "SCM_REPO:'${SCM_REPO}'"
+SCM_BRANCH = 'refs/heads/master'
+if (params.SCM_BRANCH) {
+    SCM_BRANCH = params.SCM_BRANCH
+} else if (params.ghprbPullId && ghprbGhRepository == 'eclipse/openj9') {
+    SCM_BRANCH = sha1
+}
+echo "SCM_BRANCH:'${SCM_BRANCH}'"
+SCM_REFSPEC = ''
+if (params.SCM_REFSPEC) {
+    SCM_REFSPEC = params.SCM_REFSPEC
+} else if (params.ghprbPullId && ghprbGhRepository == 'eclipse/openj9') {
```

Comment:
use `ghprbPullId` (is already initialized above at L153) instead of `params.ghprbPullId`

Reply:
Done